internal/daysteps/daysteps.go

```diff
@@ -11,10 +18,65 @@ const (
 	mInKm = 1000
 )
 
+var ErrData = errors.New("invalid data")
+var ErrSteps = errors.New("invalid steps value")
+var ErrTime = errors.New("invalid time value")
+
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+
+	dataParsed := strings.Split(data, ",")
+
+	if len(dataParsed) != 2 {
+		return 0, 0, fmt.Errorf("[parsePackage(%v)] %w: invalid data", data, ErrData)
+	}
+
+	steps, err := strconv.Atoi(dataParsed[0])
+
+	if err != nil {
+		return 0, 0, fmt.Errorf("[parsePackage(%v)] %w: invalid steps value", data, ErrSteps)
```

Comment:
Главное использовать саму ошибку, а то можешь не понять, в чем проблема
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```